### DISCUSSION TITLE
Duda general de programar en el ramo

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#7] [Ayuda] [answered]
Rengo una duda de como debiésemos programar para este ramo, ya que el enunciado de la tarea 0 está en español, pero el estándar internacional es escribir código en inglés, entonces no estaba seguro si era obligatorio/pedido programar en español o si uno puede/debe programar en inglés.

Gracias desde ya! 😃

**Comment by sachondo7** [comment, marked as answer]
Da lo mismo, tu programa a tu gusto
